atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/transaction/encoding/TicketsEncodingStrategyTest.java

```diff
@@ -160,6 +161,28 @@ public void canDecodeEmptyByteArrayAsAbortedTransaction() {
                 .isEqualTo(-1);
     }
 
+    @Test
+    public void encodeRangeOfTimestampsAsRowsIncludesAllRowsInPartitioning() {
+        assertThat(STRATEGY.encodeRangeOfStartTimestampsAsRows(2, 7)).containsExactlyElementsOf(allRowsInPartition(0));
```

Comment:
how about some edge tests here? what happens when the range is singleton, empty, or invalid (from > to)?